src/routes/+page.svelte

```diff
@@ -46,6 +46,13 @@
 		}
 	}
 
+	function deleteKnowledge(): void {
+		if (confirm('Are you sure you want to delete all knowledge?')) {
+			localStorage.removeItem('hollama-knowledge');
+			location.reload();
+		}
+	}
+
 	function deleteSettings(): void {
 		if (confirm('Are you sure you want to delete server settings?')) {
 			localStorage.removeItem('hollama-settings');
```

Comment:
Couple of things:

- `deleteSessions()`, `deleteKnoledge()` and `deleteSettings()` are identical. Let's DRY them to a single `deleteStorage(item: string)` function. 
- We should probably add a dedicated type for `item`. In `store.ts` we are dynamically generating the `localStorage` keys. Let's add an enum and use that as the type, for example: `deleteStorage(item: StorageKey)`
- These would be the values of the `StorageKey` enum: `HOLLAMA_SETTINGS: 'hollama-settings', HOLLAMA_SESSIONS: 'hollama-sessions'` and so on...